src/plugins/reporter-bugzilla.c

```diff
@@ -314,6 +314,20 @@ int main(int argc, char **argv)
 
     map_string_t *settings = new_map_string();
 
+    /* load some defaults from /etc/os-release */
```

Comment:
I the light of my recent container experience, I suggest you to first load the copy of the `/etc/os-release` file from the process' root that is stored in the problem directory under the name `os_info_in_rootdir`. Only if this first attempt to get the default Bugzilla URL for any reason (no problem directory, no such file, ...), fall back to loading of the regular `/etc/os-relase`.

Why I suggest this? Because the crashed processed might was in running in a container that were running a completely different distribution or an other version of the host distribution.

How to do that? The removals you have already done are OK and please keep them. However the additions should be amended with the following line and some more:
```
problem_data_get_osinfo(problem_data, osinfo);
```

You can find more details how to load `problem_data` here:
https://github.com/abrt/libreport/blob/master/src/plugins/reporter-bugzilla.c#L523

and how to use it here:
https://github.com/abrt/libreport/blob/master/src/plugins/reporter-bugzilla.c#L537

The `problem_data_get_osinfo` function takes care about the order of loaded `os-release` files:
https://github.com/abrt/libreport/blob/master/src/lib/problem_data.c#L640